packages/translation/src/lang/en.ts

```diff
@@ -636,6 +635,38 @@ export default {
         },
       },
     },
+    dnsHoleSummary: {
+      name: "DNS Hole Summary",
+      description: "Displays the summary of your DNS Hole",
+      option: {
+        layout: {
+          label: "Layout",
+          option: {
+            row: {
+              label: "Horizontal",
+            },
+            column: {
+              label: "Vertical",
+            },
+            grid: {
+              label: "Grid",
+            },
+          },
+        },
+        usePiHoleColors: {
+          label: "Use Pi-Hole colors",
+        },
+      },
+      error: {
+        internalServerError: "Failed to fetch DNS Hole Summary",
+      },
+      data: {
+        adsBlockedToday: "blocked today",
+        adsBlockedTodayPercentage: "blocked today",
+        dnsQueriesToday: "Queries today",
+        domainsBeingBlocked: "Domains on adlists",
```

Comment:
typo of "blocklist" ? (Might not be only ads)

Reply:
Makes more sense, yes. Thanks!